gtda/mapper/visualization.py

```diff
@@ -13,18 +11,18 @@
 
 from .utils._logging import OutputWidgetHandler
 from .utils._visualization import (
+    _validate_color_kwargs,
     _calculate_graph_data,
+    _produce_static_figure,
     _get_column_color_buttons,
     _get_colors_for_vals,
-    PLOT_OPTIONS_LAYOUT_DEFAULTS
     )
 
 
 def plot_static_mapper_graph(
-        pipeline, data, layout="kamada_kawai", layout_dim=2,
-        color_variable=None, node_color_statistic=None,
-        color_by_columns_dropdown=False, clone_pipeline=True, n_sig_figs=3,
-        node_scale=12, plotly_params=None
+        pipeline, data, color_data=None, color_features=None,
+        node_color_statistic=None, layout="kamada_kawai", layout_dim=2,
+        clone_pipeline=True, n_sig_figs=3, node_scale=12, plotly_params=None
```

Comment:
My understanding, from the docstrings, is that `node_color_statistic` super-seeds `color_data` and `color_features`, when all are passed.
Generally, there are many possible behaviors that can occur, depending on the values of those 3 parameters. While i cannot think of better names for those parameters, I was thinking about moving `node_color_statistic` forward, to stress that dependency. On the other hand, it can be seen as part of the "lower--level api", and thus, has its place further down.

Reply:
That's a good point. But the default case is applying `np.mean` to `color_features`, so reading the docs linearly it makes maybe more sense to first explain what `color_data` and `color_features` are?

Reply:
That's also a good point. And, the order that you proposed is more in line with the previous implementation.

Reply:
Ok!